pkg/gencosmosdb/generate.go

```diff
@@ -1,3 +1,3 @@
 package gencosmosdb
 
-//go:generate go-bindata -pkg gencosmosdb -prefix cosmosdb -nometadata cosmosdb
+//go:generate go run ../../vendor/github.com/go-bindata/go-bindata/go-bindata -nometadata -pkg gencosmosdb -prefix cosmosdb -nometadata cosmosdb
```

Comment:
`-nometadata` is present twice. Is this intentional?

Reply:
good catch, fixed it just now, thank you!